src/dataProvider/syncMasterDataProvider.tsx

```diff
@@ -0,0 +1,24 @@
+import { DataProvider } from "react-admin";
+
+const syncMasterDataProvider: DataProvider = {
+    getGroupList: async () => {
+        const myHeaders = new Headers();
+        myHeaders.append(
+            "Authorization",
+            "Bearer " + localStorage.getItem("token"),
+        );
```

Comment:
```suggestion
        const myHeaders = new Headers();
        const token: string | null = localStorage.getItem("token");
        if (token) {
            myHeaders.append("Authorization", "Bearer " + token);
        }
```
Если пользователь не авторизован, то пусть бэкенд считает его таковым